android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/ReplicatorMisc.java

```diff
@@ -0,0 +1,77 @@
+package com.couchbase.lite;
+
+import com.couchbase.litecore.C4Error;
+import com.couchbase.litecore.C4ReplicatorStatus;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class ReplicatorMisc extends BaseReplicatorTest {
```

Comment:
It should be `ReplicatorMiscTest`.

Reply:
Oops! I was thinking about naming ReplicatorMiscTest in mind, but my hands betrayed.